src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -333,6 +333,7 @@ internal sealed class MethodInfo
         public MethodDebugInformation DebugInformation;
         public MethodDefinitionHandle methodDefHandle;
         private MetadataReader pdbMetadataReader;
+        private bool hasDebugInformation;
```

Comment:
nit: IIUC, we can skip this, and just check `DebugInformation is not null` in the two places we use this?